pipeline/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -342,11 +363,10 @@
     "output: f'{_input:nn}.bed'\n",
     "task: trunk_workers = 1, trunk_size = job_size, walltime = walltime, mem = mem, cores = numThreads, tags = f'{step_name}_{_output:bn}'\n",
     "bash: container = container, expand= \"${ }\", stderr = f'{_output:n}.stderr', stdout = f'{_output:n}.stdout'\n",
+    "    # --vcf-half-call m  treat half-call as missing\n",
```

Comment:
please fix this comment string -- when this is uncommented the code will break.

Reply:
Can I remove this comment altogether? putting this comment anywhere else will break the command

Reply:
OK I put it in the text section above